src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -352,19 +352,38 @@ QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network,
     return res;
 }
 
+InferenceEngine::IExecutableNetworkInternal::Ptr Plugin::ImportNetwork(const std::string& modelFileName,
```

Comment:
Why is it needed? Doesn't base impl work?
```
std::shared_ptr<IExecutableNetworkInternal> IInferencePlugin::ImportNetwork(
    const std::string& modelFileName,
    const std::map<std::string, std::string>& config) {
    std::ifstream blobFile(modelFileName, std::ios::binary);

    if (!blobFile.is_open()) {
        IE_THROW(NetworkNotRead);
    }

    return ImportNetwork(blobFile, config);
}
```

Reply:
Oh, I'll remove it. Thank you so much!